global.js

```diff
@@ -0,0 +1,14 @@
+global.process = require('process');
+global.Buffer = require('buffer').Buffer;
+
+// Needed so that 'stream-http' chooses the right default protocol.
+global.location = {
+  protocol: 'file:',
+};
```

Comment:
will need to comment this out to avoid infinite url re-direct on web

(this file was just copied and pasted from `node-libs-react-native/globals` to debug